io-actions/src/fragment/fetch.js

```diff
@@ -0,0 +1,20 @@
+const fetch = require('node-fetch');
+
+async function main({ id, locale }) {
+    if (id) {
+        const response = await fetch(
```

Comment:
we need to handle possible errors here

Reply:
true, will move some of the translate cases there